### ISSUE TITLE
OpenJDK9 failure: Could not load java.lang.Object

### ISSUE THREAD

**Issue by stain** [siom79/japicmp#149]
> [ERROR] Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.9.1:cmp (default) on project commons-rdf-api: Execution default of goal com.github.siom79.japicmp:japicmp-maven-plugin:0.9.1:cmp failed: Could not load 'java.lang.Object': java.lang.Object. Please make sure that all libraries have been added to the classpath (CLASSPATH=) or try the option '--ignore-missing-classes'. -> [Help 1]

When trying to do `mvn clean install` on https://github.com/apache/incubator-commonsrdf/ with Ubuntu 16.04 x64's OpenJDK 9.

```
stain@biggiebuntu:~/src/incubator-commonsrdf$ export JAVA_HOME=/usr/lib/jvm/java-1.9.0-openjdk-amd64
stain@biggiebuntu:~/src/incubator-commonsrdf$ mvn -v
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T16:41:47+00:00)
Maven home: /home/stain/software/maven
Java version: 9-internal, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-9-openjdk-amd64
Default locale: en_GB, platform encoding: UTF-8
OS name: "linux", version: "4.4.0-38-generic", arch: "amd64", family: "unix"
```

Is it because `rt.jar` is gone?

**Comment by siom79** [comment]
japicmp is currently using javassist in version `3.20.0-GA` (the latest stable release). This release currently does not support Java 9. But according to [this site](http://www.java9countdown.xyz/) there are still 174 days left until the final release of Java 9. :smile: 

I have just seen that there is already a [RC of javassist](https://github.com/jboss-javassist/javassist/releases/tag/rel_3_21_0-java9-ea2) supporting Java 9. I will give it a try.

**Comment by siom79** [comment]
I could get a first version running with the RC:

```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout java9
mvn install
```

But there seems to be still a classpath problem when comparing japicmp with itself using the japicmp-maven-plugin. I will investigate that next.

**Comment by siom79** [comment]
Resolved with upgrade to javassist 3.22.0-GA.
